assets/queries/k8s/image_policy_webhook_not_defined/query.rego

```diff
@@ -0,0 +1,23 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.k8s as k8sLib
+
+
+CxPolicy[result] {
+	resource:= input.document[i]
+    resource.kind == "AdmissionConfiguration"
```

Comment:
Please, add `AdmissionConfiguration` in line 31, in the file `kics/.github/workflows/validate-k8s-samples.yml`

![image](https://user-images.githubusercontent.com/74001161/160583398-27e6bc68-ceee-411c-a501-dfc26702c529.png)